app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -127,14 +134,16 @@ private fun UserInputtingContent(
         DatePickerDialog(
             context,
             { _: DatePicker, Year: Int, Month: Int, DayOfMonth: Int ->
-                birthDate = "$Year-${Month}-$DayOfMonth"
+                birthDate = "$Year-$Month-$DayOfMonth"
                 val split = birthDate.split("-")
                 val year = split[0].toInt()
```

Comment:
You are accessing int indexes of an array and this can fail. You need to check the size of the array first.

Reply:
Especially because a negative number kills your "-" strategy

Reply:
You just want to be safe if any tiny thing goes wrong, which it definitely can when it doesn't seem like it

Reply:
fair enough, added an if to make sure the size is the expected 3

Reply:
made the 3 a constant as well